pandas/tests/base/test_factorize.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
 import pytest
 
+from pandas.core.dtypes.generic import ABCMultiIndex
```

Comment:
this is unnecessary in tests; the ABCFoo classes exist to avoid circular imports in the non-test code

Reply:
Ah, good to know 👍 Closing this then